iaso/models/base.py

```diff
@@ -274,11 +275,18 @@ class Task(models.Model):
 
     class Meta:
         ordering = ["-created_at"]
+        indexes = [
+            models.Index(fields=["account"]),
```

Comment:
```suggestion
```

This is not necessary because a database index is [automatically created](https://docs.djangoproject.com/en/5.1/ref/models/fields/#foreignkey) on the `ForeignKey` (unless `db_index=False` is set).

Reply:
Good point, thanks!